From 14866f296f9b54c831bbde3e779a921c37e579da Mon Sep 17 00:00:00 2001
From: Ashutosh Chauhan <hashutosh@apache.org>
Date: Wed, 14 Sep 2016 16:41:07 -0700
Subject: [PATCH 0790/1164] CDH-45814: HIVE-14762 : Add logging while removing
 scratch space (Ashutosh Chauhan via Pengcheng
 Xiong)

Signed-off-by: Ashutosh Chauhan <hashutosh@apache.org>
(cherry picked from commit f4a857ca17348cff8671e685a13ebc01b5114c0f)

Change-Id: Id697867d4c33f0518dfb9dae9cb6fd06d6c2e25f
---
 ql/src/java/org/apache/hadoop/hive/ql/Context.java |    5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/ql/src/java/org/apache/hadoop/hive/ql/Context.java b/ql/src/java/org/apache/hadoop/hive/ql/Context.java
index 9f5c54e..15ac161 100644
--- a/ql/src/java/org/apache/hadoop/hive/ql/Context.java
+++ b/ql/src/java/org/apache/hadoop/hive/ql/Context.java
@@ -91,7 +91,7 @@
   protected int tryCount = 0;
   private TokenRewriteStream tokenRewriteStream;
 
-  private String executionId;
+  private final String executionId;
 
   // List of Locks for this query
   protected List<HiveLock> hiveLocks;
@@ -367,6 +367,7 @@ public void removeScratchDir() {
       try {
         Path p = entry.getValue();
         FileSystem fs = p.getFileSystem(conf);
+        LOG.debug("Deleting scratch dir: " + p);
         fs.delete(p, true);
         fs.cancelDeleteOnExit(p);
       } catch (Exception e) {
@@ -487,6 +488,7 @@ public void clear() throws IOException {
     if (resDir != null) {
       try {
         FileSystem fs = resDir.getFileSystem(conf);
+        LOG.debug("Deleting result dir: " + resDir);
         fs.delete(resDir, true);
       } catch (IOException e) {
         LOG.info("Context clear error: " + StringUtils.stringifyException(e));
@@ -496,6 +498,7 @@ public void clear() throws IOException {
     if (resFile != null) {
       try {
         FileSystem fs = resFile.getFileSystem(conf);
+        LOG.debug("Deleting result file: " + resFile);
         fs.delete(resFile, false);
       } catch (IOException e) {
         LOG.info("Context clear error: " + StringUtils.stringifyException(e));
-- 
1.7.9.5

